### ISSUE TITLE
Lazy Loading js widgets

### ISSUE THREAD

**Issue by silicondales** [a3rev/a3-lazy-load#3]
I am somewhat motivated to roll sleeves up and contribute... I'm using this here plugin to lazy load, and have run into some .js widgets which I'd like to have this plugin cover.

Am looking to add this in for my requirements, and am forking. Would you be interested in a PR if / when I have this going?
http://ressio.github.io/lazy-load-xt/demo/widget.htm

This is an enhancement... not a bug or an issue.

**Comment by a3rev** [comment]
@silicondales Yes that would be great Rob, welcome aboard

**Comment by silicondales** [comment]
np - have been playing with the lazy load xt side, and once I have the syntax down, I'll start putting this together. Its kind of side-project to the main show, so slow but steady :)

**Comment by a3rev** [comment]
@silicondales Yes I see you have forked the Lazy Load TX script so thought you must have been familiar with it. If you need to discuss any aspect of the a3 lazy Load code just give a shout out here

**Comment by a3rev** [comment]
@robin-scott Hi Robin - have you made any progress with your PR for this issue?

**Comment by robin-scott** [comment]
Urgh no - this one we had a project which needed to lazy load some social features and a decision "up the chain" killed the social features on it... so kinda got punted for me / us @ my business. Its sitting in a Kanban board somewhere under won't fix... but if you're keen to get this rolled in, I will see if/where it can be picked up again.

**Comment by robin-scott** [comment]
@a3rev Sorry for leaving this thread hanging - it was kind of hooked to this underlying task for me.

**Comment by a3rev** [comment]
@robin-scott 

Hey Robin - thanks for filling me in and no problem, I had thought something must have happened so was just following up.

Yes itvwould be a good feature to have in a3 Laxy Load and I'm sure there would be a3 Laxy Load users who would benefit from it. But Robin  I totally understand that you and your team are busy and if it's not for a client project it has zero priority - so there is absolutely no pressure from us to do it.

If that is the case do you mind if I leave your issue open and add a [ Help Wanted ] label to it and hope someone else will pick it up?

Steve

**Comment by robin-scott** [comment]
Yeah do it - we do a lot of optimization stuff, so I could see it coming up at some stage.

**Comment by pcfreak30** [comment]
@a3rev see https://wordpress.org/plugins/rocket-footer-js/ sort have solved a lot of this :), though it would be completely incompatible with a3's code base, even though the lazy load extends a3 currently.
